aten/src/ATen/core/interned_strings.h

```diff
@@ -55,7 +55,7 @@ namespace c10 {
   _(prim, ReturnStmt)                \
   _(prim, BreakStmt)                 \
   _(prim, ContinueStmt)              \
-  _(prim, LocalVariableScope)        \
+  _(prim, ListComprehension)         \
```

Comment:
nit: ListComprehension -> ListComprehensionScope. it's not literally a ListComprehension, it's the scope that's created by one